README.md

```diff
@@ -222,6 +222,7 @@ Run the benchmark from source
 11. --partitioner: the partitioner to use in producers
 12. --jmx.servers: the jmx server addresses of the brokers 
 13. --key.distribution: Name of the distribution. Available distribution names: "uniform", "zipfian", "latest". Default: (No key)
+14. --CSV.path: A path to place the CSV file. Default: "./"
```

Comment:
It should be named as `report.path`. Also, we should have another argument to configure the format. For example: `report.format`.